go/tools/builders/cgo.go

```diff
@@ -189,6 +195,34 @@ func run(args []string) error {
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("error running cgo: %v", err)
 	}
+	// Now we fix up the generated files
+	trim := linePrefix + abs(".")
+	for _, src := range cgoOuts {
+		body, err := ioutil.ReadFile(src)
+		if err != nil {
+			return err
+		}
+		out, err := os.Create(src)
+		if err != nil {
+			return err
+		}
+		buf := bufio.NewWriter(out)
+		s := bufio.NewScanner(bytes.NewReader(body))
+		for s.Scan() {
```

Comment:
Scanner errors on long lines (over 64K IIRC). I guarantee we'll see that in the wild. You can increase the maximum length to an arbitrary limit, but I wonder if the transformation could be done with `bytes.Replace` instead?